json-endpoints/jetpack/class.jetpack-json-api-get-database-object-backup-endpoint.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+class Jetpack_JSON_API_Get_Database_Object_Backup_Endpoint extends Jetpack_JSON_API_Endpoint {
+	// /sites/%s/database-object/backup      -> $blog_id
+
+	protected $needed_capabilities = array(); // This endpoint is only accessible using a site token
+	protected $object_type;
+	protected $object_id;
+
+	// Full list of database objects that can be retrieved via this endpoint.
+	protected $object_types = array(
+		'woocommerce_attribute' => array(
```

Comment:
Should object types be something that is filterable ? This way plugins could declare support themselves?

Reply:
Sounds reasonable, but let's add it in another PR if the need comes up.